packages/@dcl/inspector/src/lib/data-layer/client/local-data-layer.ts

```diff
@@ -0,0 +1,69 @@
+import { createEngine } from '../host/engine'
+import { DataLayerRpcClient, DataLayerRpcServer, FileSystemInterface } from '../types'
+import { initRpcMethods } from '../host/rpc-methods'
+import { Composite, compositeFromJson, CompositeProvider, instanceComposite } from '@dcl/ecs'
+
+export async function createFsCompositeProvider(fs: FileSystemInterface): Promise<CompositeProvider> {
+  const compositePaths = (await fs.getDirectoryFiles('/')).filter(
```

Comment:
why we have this fn here ? I think both client/server rpc can use the same logic by only wrapping the fileSystem 🤔

Reply:
you're right, moving it :)